src/Illuminate/Foundation/Application.php

```diff
@@ -568,7 +568,7 @@ public function detectEnvironment(Closure $callback)
     public function runningInConsole()
     {
         if ($this->isRunningInConsole === null) {
-            $this->isRunningInConsole = Env::get('APP_RUNNING_IN_CONSOLE') ?? (\PHP_SAPI === 'cli' || \PHP_SAPI === 'phpdbg');
```

Comment:
Please revert these. They are there for performance reasons because `runningInConsole` is hot code in most apps.